model/trigger/version.go

```diff
@@ -149,26 +149,22 @@ func (t *versionTriggers) versionSuccess(sub *event.Subscription) (*notification
 }
 
 func (t *versionTriggers) versionRegression(sub *event.Subscription) (*notification.Notification, error) {
-	if t.data.Status != evergreen.VersionFailed {
+	if t.data.Status != evergreen.VersionFailed || t.version.Requester != evergreen.RepotrackerVersionRequester {
 		return nil, nil
 	}
 
 	versionTasks, err := task.Find(task.ByVersion(t.version.Id))
```

Comment:
I found this out yesterday, but EDN doesn't support display tasks because I've used the wrong task fetching method. Can you change this to FindWithDisplayTasks?